### PR TITLE
LPS-117364 The user icon is not shown in the Content Performance Panel

### PR DESCRIPTION
**Pull request by darquesdev**
**On hold of:**
* https://github.com/brianchandotcom/liferay-portal/pull/92029

**Previous PRs:**
* https://github.com/dgarciasarai/liferay-portal/pull/271

**Description:**
When displaying the content performance panel with a user with an associated avatar this one is never shown.

**Solution:**
Adding a couple of new properties in the display context: `authorPortraitURL` and `authorUserId`. With these Front-end is able to show the same icon as in the `UserPortraitTag`.

**How to test it**:

1. Set an image for a user and create one web content with her
2. Navigate to display page template and show the  content performance panel

Apart from that, unit and integration tests have been added ;)

**Evidences:**
Screenshot 2020-07-30 at 17 11 27 Screenshot 2020-07-30 at 17 12 36

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

File: modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java [hunk 1]
```diff
@@ -104,6 +104,19 @@ private Date _getJournalArticleFirstPublishLocalDate(
...
+	private Optional<User> _getUser(JournalArticle journalArticle) {
+		JournalArticle latestArticle =
+			_journalArticleLocalService.fetchLatestArticle(
+				journalArticle.getResourcePrimKey());
+
+		if (latestArticle == null) {
+			return Optional.empty();
+		}
+
+		return Optional.ofNullable(
```
Comment (cgoncas) [inline modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java:116]: maybe you can use  Optional.ofNullable(_journalArticleLocalService.fetchLatestArticle(journalArticle.getResourcePrimKey()).
map(
latestJournalArticle -> _userLocalService.fetchUser(latestArticle.getUserId()
). (...)

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java [hunk 2]
```diff
@@ -195,6 +197,19 @@ public String getLiferayAnalyticsURL(long companyId) {
 			"authorName",
 			_analyticsReportsInfoItem.getAuthorName(
 				_analyticsReportsInfoItemObject)
+		).put(
+			"authorPortraitURL",
+			() -> {
+				if ((_user == null) || (_user.getPortraitId() == 0)) {
```
Comment (cgoncas) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:203]: What do you think about using 
Optional.ofNullable(_user).filter (_user.getPortraitId() > 0).map(user -> user.getPortraitURL(_themeDisplay))

Comment (darquesdev) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:203]: I love it, let's stop using null checks ;)

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
@dgarciasarai back-end updated ;)

**Comment by dgarciasarai** [comment]
We have to keep this "on hold", right? Thanks! I'm going to update the frontend part :)

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
@darquesdev frontend part done! Also, I update the description :) Thanks!

**Comment by dgarciasarai** [comment]
ci:forward

**Pull request closed without merging**